src/test/java/org/folio/auth/authtokenmodule/AuthTokenTest.java

```diff
@@ -1023,4 +1042,168 @@ private String getMagicPermission(String endpoint) {
     return String.format("%s.execute", Base64.encode(endpoint));
   }
 
+  @Test
+  public void testStoreSaveRefreshToken(TestContext context) {
+    var ts = new RefreshTokenStore(vertx, tenant);
+    Async async = context.async();
+    var rt = new RefreshToken(tenant, "jones", userUUID, "http://localhost:" + port);
+    ts.saveToken(rt).onComplete(context.asyncAssertSuccess(x -> {
+      ts.checkTokenNotRevoked(rt).onComplete(context.asyncAssertSuccess(y -> {
+        async.complete();
+      }));
+    }));
```

Comment:
context.async() + async.complete() is not needed when using context.asyncAssertSuccess().
Future has built-in error handling.

```
    ts.saveToken(rt)
    .compose(x -> ts.checkTokenNotRevoked(rt))
    .onComplete(context.asyncAssertSuccess());
```